documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Technical change 1
+
+Description 1
+
+.Technical change 2
+
+Description 2
+
+[id="new-features-and-enhancements-24_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.link:https://issues.redhat.com/browse/MTV-427[MTV-427] - OCP console plugin
+
+MTV Operator installed from CLI. +
+MTV web console URL.
+
+.link:https://issues.redhat.com/browse/MTV-426[MTV-426] - OpenStack migration
+
+See new link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#rhv-prerequisites_mtv[prerequisites] for RHV and VMware migration, and link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-virtual-machines-cli_mtv[procedure] to migrate virtual machines.
+
+[id="known-issues-24_{context}"]
+== Known issues
+
+This release has the following known issues:
```

Comment:
more known issues:
1. When migrating a VM with multiple disks to more than one storage classes of type hostpath, it may result in a VM that cannot be scheduled. We recommend to use shared storage on the target OCP cluster. (for more information on this please reach out to @liranr23 )
2. When removing a VM that was migrated, its PVCs and PVs are not deleted. Workaround: remove the CDI importer pods and then remove the remaining PVCs and PVs. (https://issues.redhat.com/browse/MTV-492)
3. When archiving and deleting a migration plan that failed, its PVCs and PVs are not deleted. Workaround:  remove the CDI importer pods and then remove the remaining PVCs and PVs. (https://issues.redhat.com/browse/MTV-493)
4. VM with multiple disks that was migrated may not be able to boot on the target OCP cluster. Workaround: set the boot order appropriately to boot from the bootable disk. (https://issues.redhat.com/browse/MTV-433)
5. Warm migrations and migrations to remote OCP clusters from vSphere do not support all types of guest operating systems that are supported in cold migrations to the local OCP cluster. It is a consequence of using RHEL 8 in the former case and RHEL 9 in the latter case. See https://access.redhat.com/articles/1351473 for the list of supported guest operating systems.